l2beat/l2beat
# Add Scroll (full)

Resolves L2B-2786

# Left to do
- add RPC for Activity
- contracts diagram (will leave this to Bartek to force him to take a look at Scroll as well 😛)

# How to review
- Read the page and see if you're able to understand what's going on
- Check whether all claims are backed by references
- Check whether the sections contain all the contracts from `discovered.json`
  - "[...]Fallback" contracts have been omitted on purpose since they are dummy contracts with the same address as L2 counterparts and are there only to recover funds sent incorrectly to them. Not displayed because they are a lot and would pollute the interface a lot, and also not informative.

I also added the state derivation section but left the node software requirement as 'UnderReview' in stages because i want to discuss with Bartek the details of it.
